models/Category.js

```diff
@@ -33,6 +33,26 @@ const CategorySchema = new mongoose.Schema(
     ref: 'Subject',
     required: true
     },
+    superadmin: {
```

Comment:
Why do we have all these fields in category schema?

Reply:
Do we really need `admin`, `tutor`, `superadmin`, `subject` and `students` field for the Category model?

To what purpose do they serve?